bigchaindb/common/transaction.py

```diff
@@ -478,7 +478,7 @@ class Transaction(object):
     VERSION = '1.0'
 
     def __init__(self, operation, asset, inputs=None, outputs=None,
-                 metadata=None, version=None):
+                 metadata=None, version=None, hid=None):
```

Comment:
can you change the name of `hid` to something that describes what `hid` is? (e.g. hash_id)

Reply:
Done!